Readme: Making Request and Response types explicit in Lambda.run call…

fabianfett opened the pull request:
…back

tomerd commented:
thanks @fabianfett this is probably the right thing to do given #107. I think we should also update the examples in the Examples directories as they additional source of copy/paste 

Also, from a formatting pov, why break the signature to a separate line?

fabianfett commented:
@tomerd I've added the line breaks to ensure the callback is better visible in mobile. 

<img width="347" alt="Screen Shot 2020-06-07 at 13 22 01" src="https://user-images.githubusercontent.com/6780861/83967667-ffaf7780-a8c3-11ea-9dc1-2cb42ab02e56.png">

Removed them for now though

fabianfett commented:
@swift-server-bot test this please
